Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -174,6 +175,39 @@ void ABenchmarkGymGameModeBase::InitialiseActorCountCheckTimer()
 	}
 }
 
+void ABenchmarkGymGameModeBase::GatherWorkerConfiguration()
+{
+	// No need to fiddle with configuration as the defaults should reflect the native scenario.
+	if (!USpatialStatics::IsSpatialNetworkingEnabled())
+	{
+		return;
+	}
+
+	const UWorld* World = GetWorld();
+	const UAbstractSpatialMultiWorkerSettings* MultiWorkerSettings =
+		USpatialStatics::GetSpatialMultiWorkerClass(World)->GetDefaultObject<UAbstractSpatialMultiWorkerSettings>();
+
+	if (MultiWorkerSettings != nullptr && MultiWorkerSettings->WorkerLayers.Num() > 0)
+	{
+		NumWorkers = 0;
```

Comment:
Can just call MultiWorkerSettings->GetMinimumRequiredWorkerCount() here, it'll do the iteration internally

Reply:
POOOOG!